lxc/list.go

```diff
@@ -53,12 +55,24 @@ A single keyword like "web" which will list any instance with a name starting by
 A regular expression on the instance name. (e.g. .*web.*01$).
 A key/value pair referring to a configuration item. For those, the
 namespace can be abbreviated to the smallest unambiguous identifier.
+A key/value pair where the key is a shorthand. Multiple values must be delimited by ','. Available shorthands:
+  - type={'CONTAINER','VIRTUAL-MACHINE'}
+  - status={'RUNNING','STOPPED','FROZEN'}
+  - architecture={cpu-arch|string}
+  - device={device name|string}
+  - name={name|string}"
+  - location={location name|string}
+  - description={description|string}
+  - os={os name|string}
```

Comment:
`os` can already be done with `image.os` so I don't think it needs a shorter filter (especially as it's just an image property so there's no guarantee it will always be there)